### ISSUE TITLE
Feature Request: Visualize VsVim markers in a similar way like Visual Studio bookmarks

### ISSUE THREAD

**Issue by MartinL0815** [jaredpar/VsVim#574]
It would be really nice to have more visual feedback (like described in issue #408), but this time it's about having visual feedback not about the result of executed actions or commands, but about the position of markers.

While Visual Studio bookmarks are visualized by "blue" blocks left to the line numbers (if shown), it would be nice to have a similar visualization of the markers with e.g. "orange" blocks containing the marker "name" (a-z, A-Z, etc.).
Even the markers "<" and ">" and some of the other special markers (as supported by VsVim) should be shown.

**Comment by JohnnyPP** [comment]
This would be a really cool feature.

**Comment by Chadehoc** [comment]
I was about to suggest that, and found this request from 2013. Is it still in your basket?

**Comment by jaredpar** [comment]
It's still in my TODO list.  The feature is easy enough.  It's just getting the time to implement it :frowning:

**Comment by storypixel** [comment]
would love this

**Comment by ricksladkey** [comment]
I have a working version of this, but the PR depends on PR #2139, which hasn't been merged yet. When and if it is merged, I will submit a new PR for mark glyphs.

![image](https://user-images.githubusercontent.com/2680524/41139679-3646110c-6ab7-11e8-94ab-51c18bd88605.png)
